packages/core/src/dsn.ts

```diff
@@ -67,7 +65,7 @@ export class Dsn implements DsnComponents {
       path = split.slice(0, -1).join('/');
       projectId = split.pop() as string;
     }
-    assign(this, { host, pass, path, projectId, port, protocol, user });
+    Object.assign(this, { host, pass, path, projectId, port, protocol, user });
```

Comment:
jtr, I know we talked offline but for documentation this will break IE 10 for sure.

Reply:
Also IE11....